README.md

```diff
@@ -3,8 +3,44 @@
 If you are the kind of person who likes thinking _long-term_ and _high-impact_
 this repo is for you!
 
-**D**o **W**hat **Y**ou **L**ove is _not_ a 3-month stepping-stone project ...
+**D**o **W**hat **Y**ou **L**ove is _not_ a stepping-stone project, we're in it for the long-haul. <br />
+We are making sure that the direction we're heading in is clear to everyone who wants to know.
 
-We _very much_ want/need help on this so if you have _time_
+### Our Goal/Mission
+
+> Empower people to sustainably unlock their potential and pursue their passion through technology.
+
+So what does this encompass? Any number of things that will need to be split out into measurable goals:
++ Living a happy life
++ Financial sustainability https://github.com/dwyl/phase-two/issues/42
++ Life-long learning
++ Productivity
++ Health https://github.com/dwyl/health
++ Sustainable living https://github.com/dwyl/aquaponics
++ Living with purpose and ethics https://github.com/dwyl/hq/issues/155
++ Understanding how to be deal with today's constant distraction and information overload
```

Comment:
@markwilliamfirth please suggest the correct grammar. 
e.g: 
```
Understanding how to be deal with today's constant distraction and information overload
``` 
should be:
```
Understanding how to deal with constant distraction and information overload.
```
or even:
```
Dealing with constant distraction and information overload.
```